app/controllers/projects_controller.rb

```diff
@@ -140,6 +163,12 @@ def project_params
       params.require(:project).permit(:name, :project_access_type, :description, :tag_list, :tags)
     end
 
+    def sanitize_tags
+      return unless Flipper.enabled?(:suggest_tags)
```

Comment:
```suggestion
      return unless Flipper.enabled?(:suggest_tags, current_user)
```
Enable feature per user